AGILE/AnimatedObject.cs

```diff
@@ -1476,7 +1476,11 @@ public void Draw(Picture picture)
                             if (colourIndex != transIndex)
                             {
                                 visualPixels[screenPos] = colourIndex;
-                                priorityPixels[screenPos] = this.Priority;
+                                //  Replace the priority pixel only if the existing one is greater than 0 (unpassable line)
+                                if (priorityIndex > 0)
```

Comment:
I think this needs to be as shown below, as the original AGI interpreter is skipping all special priority values (0, 1, and 2) when drawing to the priority screen:

`if (priorityIndex > 2)`